mission_control/test/test_files/test_missions/waypoint_mission_test.xml

```diff
@@ -1,11 +1,14 @@
- <!-- This file is used for testing  -->
- 
- <root main_tree_to_execute = "MainTree" >
+<!-- This file is used for testing  -->
 
-     <BehaviorTree ID="MainTree">
-        <Sequence name="mission test">
-            <GoToWaypoint     depth="1.0" altitude="2.0" latitude="3.0" longitude="4.0" wp_radius="5.0" speed_knots="6.0" time_out="5"/>
-        </Sequence>
-     </BehaviorTree>
-
- </root>
+<root main_tree_to_execute="main" >
+  <BehaviorTree ID="main">
+    <Sequence>
+      <GoToWaypoint
+          latitude="3.0"
+          longitude="4.0"
+          altitude="2.0"
+          speed_knots="6.0"
+          tolerance_radius="0.1"/>
```

Comment:
Should we mantein the time_out port for being tested by test_if_mission_control_aborts_when_behavior_returns_time_out_failure.test?

Reply:
I was hoping we can use Timeout decorators.